set.mm

```diff
@@ -471539,6 +471657,281 @@ commutative monoid (=vectors) together with a semiring (=scalars) and a
+      n0 ex sylc simp3d ad7antr simp-4r simplr simp-8l cqus a1i eqidd wer eqger
+      simpl wbr 2idlcpbl simprl simprr ringcl qusmulval ad4antr simpllr oveq12d
+      syl211anc 3eqtr3d eqg0el syl21anc rsp2 impl imp syl1111anc eqeq2d 3bitrrd
+      eqeq1d wb wrex eleqtrrd elqsi r19.29a sylanbrc orbi12d mpbid cqs ovexd id
+      cvv qusbas anasss ralrimivva isdomn isidom ) BIJZCBUEKJZLZAIJZAUFJZAUGJUU
+      LUUMCBUHKZJZUUOUULBUIJZUUMUURBUJZCBUKULZBCAUUQDUUQMZUMUOUUNAUNJZENZFNZAUP
+      KZOZAUQKZPZUVDUVHPZUVEUVHPZURZVHZFAUSKZQEUVNQUUPUUNAUIJZGNZUVNUVHUTZVAZJZ
+      GVBZUVCUULUUSUUMCBVCKZJZUVOUUTUULUUMUURUWBUVAUULUURUWBUULUUQUWACBUUQUVBVD
+      VEVFZUOBCAUWADUWAMZVGVIZUUNUVRVJVKZUVTUUNUVQUVNVLUVQUVNVKUWFUUNUVHUVNUUNU
+      VOUVHUVNJUWEUVNAUVHUVNMZUVHMZVMRVNUUNUVQCUTZUVNUUNUVHCUULUUMUURUVHCPZUVAU
+      ULUURLZBUQKZBCVOOZSZUVHCUWKCBVPKJZUWNUVHPUULUUSUURUWOUUTBCVQULCBAUWLDUWLM
+      ZVRRUWKCBVSKJZUWNCPUULUUSUURUWQUUTBUUQCUVBVTZULZUWMBBUSKZCUWLUWTMZUWMMZUW
+      PWARWBZUOWCUUNUVNUWIUUNUVNUWICUWTUUNUURCUWTVKZUVPHNZBUPKZOZCJZUVPCJZUXECJ
+      ZURZVHZHUWTQGUWTQZUULUUMUURUXDUXMWKGHUWTCBUXFUXAUXFMZWDVFZWEUUNUVNUWIPZLZ
+      BWFJZUWQUXPCUWTPUULUXRUUMUXPUULUUSUXRUUTBWGRZWHUXQUUSUURUWQUULUUSUUMUXPUU
+      TWHUUNUURUXPUVAWIUWRTUUNUXPWLUWTABCUXADWJWMWNWOWPUVQUVNWQTGUVRXAWRUVOUVSU
+      VCGUVOUVSUVCUVNAUVPUVHUWHUWGWSXBWTXCUUNUVMEFUVNUVNUUNUVDUVNJZUVEUVNJZUVMU
+      UNUXTLZUYALZUVIUVLUYCUVILZUVDUVPUWMSZPZUVLGUWTUYDUVPUWTJZLZUYFLZUVEUXEUWM
+      SZPZUVLHUWTUYIUXEUWTJZLZUYKLZUXKUVLUYNUXMUYGUYLUXHUXKUUNUXMUXTUYAUVIUYGUY
+      FUYLUYKUUNUURUXDUXMUXOXDXEUYDUYGUYFUYLUYKXFZUYIUYLUYKXGZUYNUXRUWQUXGUWMSZ
+      CPZUXHUYNUULUXRUULUUMUXTUYAUVIUYGUYFUYLUYKXHZUXSRZUYNUULUURUWQUYSUUNUURUX
+      TUYAUVIUYGUYFUYLUYKUVAXEZUWSTZUYNUYEUYJUVFOZUYQCUYNUULUURUYGUYLVUCUYQPUYS
+      VUAUYOUYPUWKUWMBUVFUXFAUWTUVPUXEIUAUBUCUDABUWMXIOPZUWKDXJUWKUWTXKUWKUWQUW
+      TUWMXLUWSUWMBUWTCUXAUXBXMRUULUURXNUULUUSUURUWBUCNZUBNZUWMXOUDNZUANZUWMXOL
+      VUEVUGUXFOVUFVUHUXFOZUWMXOVHUUTUWCVUEVUGVUFVUHBCUXFUWMUWAUWTUXAUXBUWDUXNX
+      PVIUWKVUFUWTJZVUHUWTJZLZLUUSVUJVUKVUIUWTJUULUUSUURVULUUTWHUWKVUJVUKXQUWKV
+      UJVUKXRUWTBUXFVUFVUHUXAUXNXSWMUXNUVFMZXTYDUYNUVGUVHVUCCUYDUVIUYGUYFUYLUYK
+      UYCUVIWLYAUYNUVDUYEUVEUYJUVFUYHUYFUYLUYKYBZUYMUYKWLZYCUYNUULUURUWJUYSVUAU
+      XCTZYEWBUXRUWQLUYRUXHUWMBCUXGUXBYFVFYGUXMUYGLUYLLUXHUXKUXMUYGUYLUXLUXLGHU
+      WTUWTYHYIYJYKUYNUXIUVJUXJUVKUYNUVJUVDCPUYECPZUXIUYNUVHCUVDVUPYLUYNUVDUYEC
+      VUNYNUYNUXRUWQVUQUXIYOUYTVUBUWMBCUVPUXBYFTYMUYNUVKUVECPUYJCPZUXJUYNUVHCUV
+      EVUPYLUYNUVEUYJCVUOYNUYNUXRUWQVURUXJYOUYTVUBUWMBCUXEUXBYFTYMUUAUUBUYIUVEU
+      WTUWMUUCZJZUYKHUWTYPUYDVUTUYGUYFUYDUVEUVNVUSUYBUYAUVIXGUULVUSUVNPUUMUXTUY
+      AUVIUULUWMBAUWTUUFIVUDUULDXJUULUWTXKUULBCVOUUDUULUUEUUGYAZYQWHHUWTUVEUWMY
+      RRYSUYDUVDVUSJUYFGUWTYPUYDUVDUVNVUSUUNUXTUYAUVIYBVVAYQGUWTUVDUWMYRRYSXBUU
+      HUUIEFUVNAUVFUVHUWGVUMUWHUUJYTAUUKYT $.
+
+    $( An ideal ` I ` in the ring ` R ` is prime if and only if the factor ring
```

Comment:
Better to say "in the commutative ring", and similarly in some theorems above which assume `R e. CRing`.